dev-libs/boehm_gc/boehm_gc-7.6.4.recipe

```diff
@@ -0,0 +1,82 @@
+SUMMARY="The Boehm-Demers-Weiser conservative garbage collector"
+DESCRIPTION="The Boehm-Demers-Weiser conservative garbage collector can be \
+used as a garbage collecting replacement for C malloc or C++ new. It allows \
+you to allocate memory basically as you normally would, without explicitly \
+deallocating memory that is no longer useful. The collector automatically \
+recycles memory when it determines that it can no longer be otherwise \
+accessed."
+HOMEPAGE="http://www.hboehm.info/gc/"
+COPYRIGHT="1988, 1989 Hans-J. Boehm, Alan J. Demers
+	1991-1996 Xerox Corporation.
+	1996-1999 Silicon Graphics.
+	1999-2011 Hewlett-Packard Development Company, L.P."
+LICENSE="BOEHM"
+REVISION="2"
```

Comment:
```diff
-LICENSE="BOEHM"
+LICENSE="Boehm"
-REVISION="2"
+REVISION="1"
```
You also need to "attach" a copy of the `Boehm` (license) file to this recipe (in `haikuports/dev-libs/boehm_gc/licenses/`).
(That's not required by very popular licenses, but this one is not yet "available" in a default Haiku install.)

Reply:
Hmm. BOEHM is already used by 7.6.0 recipe. Should I rename it (to Boehm) in both recipes?

Reply:
Sorry, I had not noticed the file was already there.
I guess it would be OK to keep it as `BOEHM`.

But maybe it would be better to change it to `Boehm` and rename the file.
Let's see what the other contributors think about this.
(I'd say that this is not an acronym, so it should probably be changed to `Boehm`, but I'm OK with keeping it as is.)